types.go

```diff
@@ -200,6 +255,10 @@ type ItemCreateParams struct {
 	Tags []string `json:"tags,omitempty"`
 	// The websites used for autofilling for items of the Login and Password categories.
 	Websites []Website `json:"websites,omitempty"`
+	// The item's files stored as fields
+	Files []FileCreateParams `json:"files,omitempty"`
+	// The document file for the Document item category
+	Document *DocumentCreateParams `json:"document,omitempty"`
```

Comment:
I'd make it super clear in the docs that this field is empty when the Item is not a document. Also should we stop using the category terminology in favour of the "item type" one?

```suggestion
	// The document file for the Document item type. Empty when the item isn't of Document type.
	Document *DocumentCreateParams `json:"document,omitempty"`
```

Reply:
I think we can keep the category terminology for now as we would have to update the rest of the terminology so we can do that than :)